pkg/minikube/cluster/cluster_test.go

```diff
@@ -50,3 +53,52 @@ func TestLocalBoot2DockerURL(t *testing.T) {
 		t.Fatalf("Expected URL : %s", localISOUrl)
 	}
 }
+
+func TestHostGetLogs(t *testing.T) {
```

Comment:
I think we discussed this before, I am not so sure of the usefulness of the test. At the very least we should make explicit what this actually does. It verifies that the '-f' flag if specified gets passed to the VM (via ssh). This is by no means a test to verify whether streaming the logs actually works or not.